### ISSUE TITLE
Allow bundling non-Lua files

### ISSUE THREAD

**Issue by gnois** [openresty/opm#79]
I have a `stops_en.txt` text file that lists stop-words to omit when generating slug. This text file will be read once during init_by_lua.

When I do `opm build`, seems like the text file is ignored. Is it because only Lua files are allowed?

Is so, can we enable bundling non-Lua files such as .txt? Perhaps with an `include_files` key in dist.ini?

**Comment by doujiang24** [comment]
yeah, `.txt` suffix files will be ignored. I think `include_files` is a good idea, will you please contribute a PR for it?

**Comment by agentzh** [comment]
I think a `MANIFEST` file similar to Perl's CPAN module is more convenient.

**Comment by gnois** [comment]
I have to admit I am not familiar with Perl.  If we go down the `MANIFEST` path, there is also `MANIFEST.SKIP` as I just learned from online. Is that going to overlap `exclude_files` key in dist.ini in terms of functionality?
